playbooks/roles/nginx/templates/assets.j2

```diff
@@ -10,6 +10,12 @@ server {
     set $communications_s3_url "{{ communications_s3_url }}";
     set $submissions_s3_url "{{ submissions_s3_url }}";
 
+    # host must be unset for S3 requests
+    proxy_set_header Host "";
```

Comment:
thats a good point - by setting, say, `proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;` we should ignore the earlier proxy_set_header calls, and leave Host untouched from the client. i'll try and test it out locally